engine/src/main/java/org/terasology/logic/ai/HierarchicalAISystem.java

```diff
@@ -125,17 +125,17 @@ private void loop(EntityRef entity, LocationComponent location,
 
             // if our AI is aggressive or hunter go and hunt player else run away
             // if wild
-            if (ai.aggressive) {
+            /*if (ai.aggressive) {
```

Comment:
Probably it was me slowly tearing out aging pieces originally introduced by Miniion and/or the ancient gelcubes but superseded but later changes. I don't think the functionality is used any more and can be removed rather than commented out.